src/sdl2/event.rs

```diff
@@ -709,6 +734,26 @@ impl ::std::fmt::Debug for Event {
     }
 }
 
+/// Helper function to make converting scancode's
```

Comment:
Bit of a trivial issue, but it should read 'scancodes and keycodes' rather than 'scancode's and keycode's'